_episodes_rmd/07-genomic-ranges.Rmd

```diff
@@ -371,7 +371,7 @@ both ranges in the subject set.
 ## The GRanges class
 
 Having defined integer ranges, the only additional information necessary to
-define genomic ranges is the name the genomic sequence on which each range is
+define genomic ranges, is the name of the genomic sequence on which each range is
```

Comment:
I agree with "of", but I don't think the "," is needed.

Reply:
True, I think I added it to sort of "break up" the long sentence, but now that I re-read it, it's indeed not strictly necessary.